product_stock_state/views/product_template_view.xml

```diff
@@ -4,10 +4,10 @@
         <field name="model">product.template</field>
         <field name="inherit_id" ref="stock.view_template_property_form" />
         <field name="arch" type="xml">
-            <field name="route_ids" position="before">
```

Comment:
I think it would be better to inherit from the base product view as stock does and hook to the same place:

` <xpath expr="//group[@name='group_lots_and_weight']" position="after">`

See https://github.com/odoo/odoo/blob/a7f7233e0eae8ee101d745a9813cba930fd03dcb/addons/stock/views/product_views.xml#L124

Reply:
good idea, thanks